main.go

```diff
@@ -79,16 +79,16 @@ func main() {
 	router.HandleFunc("/"+mnHealth, t.promHealthCheckHandler).Methods("GET")
 
 	// Path-based  multi-origin support - no support for full proxy of the prometheus UI, only querying
-	router.HandleFunc("/{originMoniker}"+prometheusAPIv1Path+mnQueryRange, t.promQueryRangeHandler).Methods("GET")
-	router.HandleFunc("/{originMoniker}"+prometheusAPIv1Path+mnQuery, t.promQueryHandler).Methods("GET")
-	router.PathPrefix("/{originMoniker}" + prometheusAPIv1Path).HandlerFunc(t.promFullProxyHandler).Methods("GET")
+	router.HandleFunc("/{originMoniker}"+prometheusAPIv1Path+mnQueryRange, t.promQueryRangeHandler).Methods("GET", "POST")
+	router.HandleFunc("/{originMoniker}"+prometheusAPIv1Path+mnQuery, t.promQueryHandler).Methods("GET", "POST")
+	router.PathPrefix("/{originMoniker}"+prometheusAPIv1Path).HandlerFunc(t.promFullProxyHandler).Methods("GET", "POST")
```

Comment:
Only the `query` and `query_range` endpoints got `POST` support added, so I think the `promFullProxyHandler` doesn't need it?